### PR TITLE
Kuma report, January 2017

### PR DESCRIPTION
**Pull request by jwhitlock**
This is the monthly Kuma report I've been posting to the [dev-mdn mailing list](https://groups.google.com/forum/#!forum/mozilla.dev.mdn). If it is appropriate for MozMEAO, I can start posting here, or can continue posting to the mailing list.

### CODE REVIEW

**Comment by pmac** [comment]
I like it! I've not scoured it by any means, but I think this use of the blog is good. 👍

File: _posts/2017-02-01-kuma-report.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+---
+Stephanie Hobson (shobson) completed the conversion of the functional tests
+from Intern to py.test with the
+[translation tests](https://github.com/mozilla/kuma/pull/4102), and has
+submitted the pull request to remove the
+[Intern tests](https://github.com/mozilla/kuma/pull/4099). We now have
+a library of browser-based functional tests to verify that an integrated
+environment is serving MDN correctly. Giorgos Logiotatidis (giorgos) has
+[re-written the Jenkins integration pipeline](https://github.com/mozilla/kuma/pull/4102)
+for both Bedrock and MDN, providing a framework for automated acceptance
```
Comment (glogiotatidis) [inline _posts/2017-02-01-kuma-report.md]: I only did MDN, @pmac did bedrock

Comment (jwhitlock) [inline _posts/2017-02-01-kuma-report.md]: I removed the reference to Bedrock

**Comment by jgmize** [comment]
Also thanks for updating @glogiotatidis :)

**Pull request merged**